Makefile

```diff
@@ -17,6 +17,10 @@ audit:
 build:
 	go build -tags 'production' $(LDFLAGS) -o $(BINPATH)/dp-sitemap
 
+.PHONY: build-cli
+build-cli:
+	go build -tags 'production' $(LDFLAGS) -o $(BINPATH)/dp-sitemap-cli cmd/cli-tool/*.go
+
```

Comment:
would suggest to have two commands - one for local running and one for remote. Different tags and GOOS etc. 

Include this in the README for the CLI tool as well.